src/R/Support/Impl/Help/Packages/UserPackagesCollection.cs

```diff
@@ -33,8 +35,9 @@ public sealed class UserPackagesCollection : PackageCollection {
                     } else {
                         // Multiple library folders. Try and match to R version
                         // specified in the Tools | Options
-                        string version = GetReducedVersion();
-                        string path = Path.Combine(libraryPath, version);
+                        var version = RInstallation.GetRVersion(RToolsSettings.Current.RBasePath);
+                        var versionString = Invariant($"{version.Major}.{version.Minor}");
```

Comment:
In RTM we'll go asking R rather than figuring this manually. But at least we not get version # from file rather than from disk path.